pom.xml

```diff
@@ -117,6 +117,11 @@
       <artifactId>plexus-container-default</artifactId>
       <version>1.0-alpha-9</version>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.11</version>
```

Comment:
This gives us the `DigestUtils.md5Hex( inputStream )` method used to compare if two .class files are the same.